handlers/developers_handler.js

```diff
@@ -47,27 +14,110 @@ var DeveloperHandler = {
     this.client = redis.createClient({password: process.env.REDIS})
   },
   /**
+   * Set the response object to be in the format of a successful reponse
+   * @param {object} res - The response object
+   * @param {string} username - The given username
+   * @param {object} token - The given testing-token
```

Comment:
Ideally, we do not use `{object}` when specifying a type, since it can mean anything. Since a `Token` is your own type, define it above with a jsdoc `@typedef` block (e.g. https://bitbucket.ciena.com/projects/BP_UI_APPS/repos/frost-orchestrate-ui/browse/app/utils.js#8) and then specify that type here.